src/jv.c

```diff
@@ -1636,8 +1642,8 @@ static jv jvp_object_unshare(jv object) {
     struct object_slot* new_slot = jvp_object_get_slot(new_object, i);
     *new_slot = *old_slot;
     if (jv_get_kind(old_slot->string) != JV_KIND_NULL) {
-      new_slot->string = jv_copy(old_slot->string);
-      new_slot->value = jv_copy(old_slot->value);
+      new_slot->string = jv_unborrow(old_slot->string);
```

Comment:
The keys and values stored in objects should always be owned, not borrowed, so no need to unborrow here.